src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
```

Comment:
Do we need to add property `hidden` as an argument to the constructor?
The hidden flag must always be `false` on creation
```suggestion
  constructor(name, health = 100) {
```

Reply:
fixed